nnpdf_data/nnpdf_data/new_commondata/PHENIX-2009_SHP_200GEV/uncertainties.yaml

```diff
@@ -0,0 +1,50 @@
+definitions:
+  stat:
+    description: Statistical uncertainty
+    treatment: ADD
+    type: UNCORR
+  sys_lumi:
+    description: Systematic uncertainties due to luminosity
+    treatment: MULT
+    type: CORR
+  sys_pol:
```

Comment:
By looking at the caption of Table VI of https://arxiv.org/pdf/1402.6296, I think we need to add the lumi uncertainty: 4.2 * 10^-4, correlated additive. ( while we are not considering the polarization scale uncertainty, 4.8%).

Reply:
Why additive?

Reply:
I think we treat the lumi unc as additive (?)

Reply:
Usually, unless explicitly stated otherwise, luminosity uncs are treated as multiplicative.